drivers/hmis_external_apis/app/models/hmis_external_apis/external_forms/config.rb

```diff
@@ -21,23 +21,6 @@ def initialize
       return if Rails.env.development? || Rails.env.test?
 
       raise 'Missing AppConfigProperty for presign url' if presign_url.blank?
-      raise 'Missing submission bucket' if submission_s3_url.blank?
-    end
-
-    def submission_s3_url
-      GrdaWarehouse::RemoteCredentials::S3.for_active_slug('public_bucket')&.s3&.bucket&.url
-    end
-
-    def csp_content
```

Comment:
this would now need to be set as config with the key "external_forms/csp_content"